### DISCUSSION TITLE
ZigBee: Is it possible to configure bettry powered device timeout?

### DISCUSSION THREAD

**Discussion by drevster** [arendst/Tasmota#10775] [Q&A] [answered]
Hi,
i use ZigBee Bridge connected to "Home Assistant" using ZHA. 

i did a test and took a battery from lumi.weather sensor. This device remain available in Home Assistant for 6 hours and later became unavailable (offline). i wonder if this timout is configurable or even a hardcoded value?

**Comment by ascillato** [comment, marked as answer]
Sorry, not supported.
